plugins/browser-plugin-form-tracking/src/helpers.ts

```diff
@@ -252,17 +228,20 @@ function getInnerFormElements(trackingMarker: string, elt: HTMLFormElement) {
       if (child.type === 'submit') {
         return;
       }
-      var elementJson: ElementData = {
-        name: getElementIdentifier(child),
-        value: child.value,
-        nodeName: child.nodeName,
+      var elementJson: ElementDataWrapper = {
+        elementData: {
```

Comment:
@adatzer isn't this a breaking change since we changed the structure from 
`{ name, value, nodeName } to { elementData: { name, value, nodeName }, originalElement }` ?

Reply:
This type is not exported. Both `ElementData` and the transform function's arguments haven't changed.